src/components/LocationPage/Notifications.tsx

```diff
@@ -48,12 +48,19 @@ const NotificationArea: React.FC<{ projections: Projections }> = ({
     HospitalizationsPeak,
     ThirdWave,
     ExposureNotifications,
+    TXFeb2021Winter,
     None,
   }
   let notification: Notification;
 
   if (showExposureNotifications(region)) {
     notification = Notification.ExposureNotifications;
+  } else if (
+    // TODO(2021/2/20): TX Winter Reporting
+    // https://trello.com/c/TdspuIeM/952-texas-reporting-dip-winter-weather-feb-2020
+    region.fipsCode.startsWith('48')
```

Comment:
This breaks in the face of MSA's, e.g. Watertown-Fort Drum, NY has code 48060.  I'll make a fix and push it to this branch.

Reply:
Fixed.